dcctools/exact_matches.py

```diff
@@ -8,12 +8,20 @@
     description="Tool for counting the number of exact matches (exactly identical hashes) \
                  for one project across sites."
 )
-parser.add_argument(
+group = parser.add_mutually_exclusive_group()
+group.add_argument(
     "-p",
     "--project",
     default="name-sex-dob-zip",
     help="Select a project to review exact matches across sites. \
-          Default: name-sex-dob-zip",
+          Default: name-sex-dob-zip \
+          Does not work with the household flag",
+)
+group.add_argument(
+    "--household",
+    help="Review exact household matches. \
+          Default: fn-phone-addr-zip \
+          Does not work with the project flag",
```

Comment:
I'm not seeing a good way to actually make the default fn-phone-addr-zip* so I would suggest changing this text (and using the above as the template for consistency):

```suggestion
    help="Select a project to review exact household matches across sites. \
          Does not work with the project flag",
```

* = yes you can set the `default="fn-phone-addr-zip",` as above, but then it's always enabled by default which we don't want